runbotics/runbotics-orchestrator-ui/src/src-app/Actions/ExcelActions/deleteRows.ts

```diff
@@ -0,0 +1,61 @@
+import { ExcelAction } from 'runbotics-common';
+
+import { translate } from '#src-app/hooks/useTranslations';
+
+import { ActionSystem, IBpmnAction, Runner } from '../types';
+const deleteRows = (): IBpmnAction => (
+    {
+        id: ExcelAction.DELETE_ROWS,
+        label: translate('Process.Details.Modeler.Actions.Excel.DeleteRows.Label'),
+        script: ExcelAction.DELETE_ROWS,
+        runner: Runner.DESKTOP_SCRIPT,
+        system: ActionSystem.WINDOWS,
+        output: {
+            assignVariables: true,
+            outputMethods: {
+                variableName: '${content.output[0]}',
+            },
+        },
```

Comment:
neither delete columns, nor RPA framework returns deleted rows/cols
imo usecase for that occurs hardly ever